plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -0,0 +1,1393 @@
+	 * Outputs the text to display when there are no reviews to display.
+	 *
+	 * @see WP_List_Table::no_items()
+	 *
+	 * @global string $comment_status
+	 *
+	 * @return void
+	 */
+	public function no_items() {
+		global $comment_status;
+
+		if ( 'moderated' === $comment_status ) {
+			esc_html_e( 'No reviews awaiting moderation.', 'woocommerce' );
+		} else {
+			esc_html_e( 'No reviews found.', 'woocommerce' );
+		}
+	}
+
+	/**
+	 * Renders the checkbox column.
+	 *
+	 * @param WP_Comment $item Review or reply being rendered.
+	 * @return void
+	 */
+	protected function column_cb( $item ) {
+
+		ob_start();
+
+		if ( $this->current_user_can_edit_review ) {
+			?>
+			<label class="screen-reader-text" for="cb-select-<?php echo esc_attr( $item->comment_ID ); ?>"><?php esc_html_e( 'Select review', 'woocommerce' ); ?></label>
+			<input
+				id="cb-select-<?php echo esc_attr( $item->comment_ID ); ?>"
+				type="checkbox"
+				name="delete_comments[]"
+				value="<?php echo esc_attr( $item->comment_ID ); ?>"
+			/>
+			<?php
+		}
+
+		echo $this->filter_column_output( 'cb', ob_get_clean(), $item ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Renders the review column.
+	 *
+	 * @see WP_Comments_List_Table::column_comment() for consistency.
+	 *
+	 * @param WP_Comment $item Review or reply being rendered.
+	 * @return void
+	 */
+	protected function column_comment( $item ) {
+
+		$in_reply_to = $this->get_in_reply_to_review_text( $item );
+
+		ob_start();
+
+		if ( $in_reply_to ) {
+			echo $in_reply_to . '<br><br>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		}
+
+		printf(
+			'%1$s%2$s%3$s',
+			'<div class="comment-text">',
+			get_comment_text( $item->comment_ID ), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			'</div>'
+		);
+
+		if ( $this->current_user_can_edit_review ) {
+			?>
+			<div id="inline-<?php echo esc_attr( $item->comment_ID ); ?>" class="hidden">
+				<textarea class="comment" rows="1" cols="1"><?php echo esc_textarea( $item->comment_content ); ?></textarea>
+				<div class="author-email"><?php echo esc_attr( $item->comment_author_email ); ?></div>
+				<div class="author"><?php echo esc_attr( $item->comment_author ); ?></div>
+				<div class="author-url"><?php echo esc_attr( $item->comment_author_url ); ?></div>
+				<div class="comment_status"><?php echo esc_html( $item->comment_approved ); ?></div>
+			</div>
+			<?php
+		}
+
+		echo $this->filter_column_output( 'comment', ob_get_clean(), $item ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Gets the in-reply-to-review text.
+	 *
+	 * @param WP_Comment $reply Reply to review.
+	 * @return string
+	 */
+	private function get_in_reply_to_review_text( $reply ) {
+
+		$review = $reply->comment_parent ? get_comment( $reply->comment_parent ) : null;
+
+		if ( ! $review ) {
+			return '';
+		}
+
+		$parent_review_link = esc_url( get_comment_link( $review ) );
```

Comment:
```suggestion
		$parent_review_link = get_comment_link( $review );
```

This is already escaped on line 979.

Reply:
Confirmed ✅